Source/WebCore/rendering/AttachmentLayout.mm

```diff
@@ -47,6 +47,8 @@
 const CGFloat attachmentIconBackgroundRadius = 3;
 const CGFloat attachmentIconToTitleMargin = 2;
 
+const CGFloat attachmentImageOnlyIconSize = 52;
```

Comment:
can these be constexpr?

Reply:
oh maybe because its CGFloat and not normal float it can't be? Not sure

Reply:
`CGFloat` is just a typedef `float` or `double` depending on the architecture.

Reply:
There's a mixture of `const`s and `constexpr`s, happy to change them all to the latter. 👍